src/nvim/menu.c

```diff
@@ -62,8 +62,8 @@ ex_menu (
   int modes;
   char_u      *map_to;
   int noremap;
-  int silent = FALSE;
-  int special = FALSE;
+  char silent = FALSE;
+  char special = FALSE;
```

Comment:
The value is later assigned to `silent` variable in `vimmenu_T` which is of type `char`, so I assumed it would be the correct type to use. But looking at it again, it seems it could be changed to boolean.